python/create_sandbox/dialog.py

```diff
@@ -12,7 +12,7 @@
 import datetime
 import os
 from sgtk.platform.qt import QtCore, QtGui
-from tank_vendor import six
+from tank_vendor import sgutils
```

Comment:
Does that requires a specific version of tk-core? If so, it needs to be set in info.yml - requires_core_version.

But can we have a transitional step instead? Something that could work with both old and new versions of tk-core?